securebanking-openbanking-uk-rcs-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rcs/api/dto/consent/details/DomesticStandingOrderPaymentConsentDetails.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright © 2020 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.securebanking.openbanking.uk.rcs.api.dto.consent.details;
+
+import com.forgerock.securebanking.common.openbanking.uk.forgerock.datamodel.account.FRBankAccountWithBalance;
+import com.forgerock.securebanking.openbanking.uk.common.api.meta.IntentType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.joda.time.DateTime;
+import uk.org.openbanking.datamodel.account.OBStandingOrder6;
+import uk.org.openbanking.datamodel.payment.OBActiveOrHistoricCurrencyAndAmount;
+
+import java.util.List;
+
+/**
+ * Models the consent data for a domestic standing order.
+ */
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class DomesticStandingOrderPaymentConsentDetails extends ConsentDetails {
+
+    private OBStandingOrder6 standingOrder;
+    
+    private OBActiveOrHistoricCurrencyAndAmount instructedAmount; 
```

Comment:
I've just noticed these objects are using a mixture of OB and FR objects. I'd prefer if they used the latter for reasons I've explained before. I'll aim to change this as it's quick and easy and shouldn't impact the UI (since the field names are the same in the equivalent FR objects).